src/components/containers/PlotlyPanel.js

```diff
@@ -123,23 +124,23 @@ class Panel extends Component {
   }
 }
 
-Panel.plotly_editor_traits = {no_visibility_forcing: true};
+PlotlyPanel.plotly_editor_traits = {no_visibility_forcing: true};
 
-Panel.propTypes = {
+PlotlyPanel.propTypes = {
   children: PropTypes.node,
   addAction: PropTypes.object,
```

Comment:
I left the addAction on `<Panel/>` maybe devs would like to add some non context related addActions..
Doesn't have to be disabled no?